tests/periph_timer/tests/01-run.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2016 Kaspar Schleiser <kaspar@schleiser.de>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+
+sys.path.append(os.path.join(os.environ['RIOTBASE'], 'dist/tools/testrunner'))
+import testrunner
+
+
+def testfunc(child):
+    child.expect('Available timers: \d+')
+    timers_num = int(child.match.group(0).split(' ')[-1])
```

Comment:
Why did you decide against proper regex grouping?

Reply:
mean: `re` has to parse the string anyway so why not make use of that?

Reply:
If you think 

```python
child.expect('Available timers: (\d+)')
timers_num = int(child.match.group(1))
```

is not readable enough there is also annotate the group ;-)

```python
child.expect('Available timers: (?P<timers_num>\d+)')
timers_num = int(child.match.group('timers_num'))
```

Reply:
I have to admit that I was not aware of regexp grouping. At least I learnt something today :). I applied the first solution you proposed @miri64

Reply:
Works with POSIX tools like `sed` also ;-) e.g.

```bash
git grep "gnrc_netif2" | xargs sed 's/\(gnrc_netif\)2/\1/g'
```

is really convenient ;-)